Data/data.js

```diff
@@ -0,0 +1,54 @@
+const results = [
+    {
+      id: 1,
```

Comment:
You guys did not need to have an id key in this array. You guys did not seem to use it in the app.js so it can be deleted.